Updating Integrations section of Docs (#2003)
getsentry/sentry-docs: src/docs/product/integrations/bitbucket/index.mdx

  - /workflow/global-integrations/integrations/bitbucket/
---

Track and resolve bugs faster by using data from your BitBucket commits. 

## Install

<Alert title="Note" level="info">
  Sentry owner or manager permissions, and Bitbucket admin permissions are
  required to install this integration.
</Alert>

> 1. In Sentry, navigate to **Organization Settings > Integrations**.

Reviewer: Settings > Integrations for consistency?

> 2. If you have the legacy Bitbucket integration installed, you’ll see a button next to Bitbucket that says **Upgrade**. If you do not have the legacy Bitbucket integration installed, you’ll see a button that says **Install**.

Reviewer 2: ```suggestion
2. If you have the legacy Bitbucket integration installed, use the button next to Bitbucket to **Upgrade**. If you do not have legacy Bitbucket integration installed, use the button to **Install**.
```
